src/hydra/dataProvider.js

```diff
@@ -215,11 +215,13 @@ export default (
         const {
           pagination: { page, perPage },
           sort: { field, order },
+          customParam: { value },
```

Comment:
It would be better if the custom param:
- could be applied for all methods;
- could contain multiple values